fmp_server.py

```diff
@@ -38,7 +38,22 @@ def recvAnsConnection(self, packet_id, data, seq):
         connection_data = pati.ConnectionData.unpack(data)
         self.server.debug("Connection: {!r}".format(connection_data))
         self.sendNtcLogin(3, connection_data, seq)
-
+    
+    def recvReqShut(self, packet_id, data, seq):
+        """ReqShut packet.
+
+        ID: 60100100
+        JP: 切断要求
+        TR: Disconnection request
+        """
+        # 1: full logout(?)
```

Comment:
This can be added in the docstring directly.
```python
        """ReqShut packet.
        ID: 60100100
        JP: 切断要求
        TR: Disconnection request

        Login type:
         - 0x01: full logout(?)
         - 0x02: logout to different server(?)
        """
```

Reply:
Fixed in latest commit.